src/nrnpython/CMakeLists.txt

```diff
@@ -238,7 +243,8 @@ if(NRN_ENABLE_MODULE_INSTALL)
         ${CMAKE_COMMAND} -E copy_if_different
         ${PROJECT_SOURCE_DIR}/share/lib/python/neuron/help_data.dat
         ${CMAKE_CURRENT_BINARY_DIR}/lib/python/neuron/help_data.dat
-      COMMAND ${extra_env} ${pyexe} setup.py --quiet build --build-lib=${NRN_PYTHON_BUILD_LIB}
+      COMMAND ${extra_env} ${extra_env_distutils} ${pyexe} setup.py --quiet build
```

Comment:
I think we're picking up two levels of `${CMAKE_COMMAND} -E env` now, which might re-break things with TSan (that's not in the CI yet).

How about
```cmake
set(extra_env ${CMAKE_COMMAND} -E env "SETUPTOOLS_USE_DISTUTILS=stdlib")
if(NRN_SANITIZERS)
  list(APPEND extra_env "LDCSHARED=..." "LDCXXSHARED=...")
endif()
```
instead.

Reply:
right, will do. Thanks